src/client/components/MovieList.jsx

```diff
@@ -8,7 +8,14 @@ export default function MovieList() {
   const [movies, setMovies] = useState([])
 
   useEffect(() => {
-    fetch(`${apiUrl}/movie`)
+    const options = {
+    method: "GET",
```

Comment:
remember that a fetch request is a GET request by default. You probably don't need to specify GET as a method here

Reply:
True, wasn't sure if I had to since I was being more specific in my header